guides/v2.3/config-guide/prod/config-reference-gitignore.md

```diff
@@ -1 +1,179 @@
-../../../v2.2/config-guide/prod/config-reference-gitignore.md
```

Comment:
This is a symlink file. Replacing the contents of this file will break the build process.

To properly update this version of the file, please delete the symlink file and create a new file with the updated content.

Reply:
Sorry, did not notice that. Fixed!